src/routes/boards.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const boardsData = require('../data/boards');
+
+/**
+ * Render base page that will show a user their boards
+ */
+router.get('/', async (req, res) => {
+    try {
+        //check if logged in
+        if(req.session.user) {
+            let userId = req.session.user.id;
```

Comment:
This would be req.session.user._id

Reply:
makes sense, wasn't sure how we wanted to do that